pkg/Microsoft.Private.PackageBaseline/packageIndex.json

```diff
@@ -4980,7 +4979,6 @@
       "InboxOn": {
         "netcoreapp2.0": "4.1.1.0",
         "netcoreapp2.1": "4.3.0.0",
-        "netcoreapp2.2": "4.3.0.0",
```

Comment:
Why was this deleted ?

Reply:
I am not sure why this got deleted. These changes appeared as I ran "msbuild /t:UpdatePackageIndex" command. Looking forward to @weshaggard for more guidance on this.

Reply:
I would expect to see the SqlClient version bumped in this file. I'm not entirely sure why these 2 entries were removed but I suspect it was because they were redundant. @ericstj  @joperezr?

Reply:
Yeah, they are redundant.

Reply:
>Is that what you did here @weshaggard?

I did a couple iterations at one point I generated them with the package index update task but I may have manually changed them later for these too. At any rate @AfsanehR these changes are safe but I still don't see the SqlClient version updated like I would expect. I would expect to see https://github.com/AfsanehR/corefx/blob/577778967bf657d325bd999d93dc62458e2dc0a8/pkg/Microsoft.Private.PackageBaseline/packageIndex.json#L1254 updated to the new version. Once you build the library with the new version can you again try to run the UpdatePackageIndex target on the pkgproj for it SqlClient?

Reply:
@weshaggard Running build -allConfigurations, I am getting this [error](https://ci3.dot.net/job/dotnet_corefx/job/master/job/windows-TGroup_netfx+CGroup_Release+AGroup_x86+TestOuter_false_prtest/12257/console):
``` D:\j\workspace\windows-TGrou---2a8f9c29\Tools\partialfacades.task.targets(64,5): error : Errors were encountered when generating facade(s). [D:\j\workspace\windows-TGrou---2a8f9c29\src\System.Data.SqlClient\src\System.Data.SqlClient.csproj]``` Do you have  any suggestion how to fix this?